src/background.ts

```diff
@@ -31,6 +31,16 @@ function openTab(url: string) {
         });
 }
 
+chrome.runtime.onMessageExternal.addListener(
+    function(request, sender, sendResponse) {
+        if (request) {
+            if (request.message == "ping") {
+                sendResponse({ping: true});
+            }
+        }
+    }
+);
```

Comment:
Let's wrap this into a method, listenToMessages ,and call it inside the constructor of  ShortExt

Reply:
Can we use enum for the message?